todo_app/data/trello_helper.py

```diff
@@ -31,9 +31,20 @@ def __init__(self):
 
     def set_list_ids(self):
         if self.card_lists:
-            self.todo_list_id = self.card_lists[self._LIST_TO_DO].id
-            self.doing_list_id = self.card_lists[self._LIST_DOING].id
-            self.done_list_id = self.card_lists[self._LIST_DONE].id
+            if self._LIST_TO_DO in self.card_lists:
+                self.todo_list_id = self.card_lists[self._LIST_TO_DO].id
+            else:
+                self.todo_list_id = "TO_DO_LIST_ID"
```

Comment:
I'm not sure it's neater, but worth noting that you can use the `.get` method on a dictionary as a safe alternative when you're not sure if the key exists, and can specify a default value if it misses:
```suggestion
            self.todo_list_id = self.card_lists.get(self._LIST_TO_DO, {"id": "TO_DO_LIST_ID"}).id
```